### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by pvl** [kamalkraj/BERT-NER#17]
Because of the syntax error I had to change this line from a list

output = [word:{"tag":label,"confidence":confidence} for word,label,confidence in zip(words,labels,logits_confidence)]

to a dictionary

output = {word:{"tag":label,"confidence":confidence} for word,label,confidence in zip(words,labels,logits_confidence)}

**Comment by kamalkraj** [comment]
https://github.com/kamalkraj/BERT-NER/issues/14
